pom.xml

```diff
@@ -478,7 +478,7 @@
     <version.com.sun.grizzly>1.9.59</version.com.sun.grizzly>
     <version.org.codehaus.jettison>1.4.0</version.org.codehaus.jettison>
     <version.junit>4.13.1</version.junit>
-    <version.org.jboss.byteman>4.0.13</version.org.jboss.byteman>
+    <version.org.jboss.byteman>4.0.19</version.org.jboss.byteman>
```

Comment:
This would put 5.11 ahead of `master` so I have raised https://github.com/jbosstm/narayana/pull/2050